guidelines/content-guidelines/02-diagrams.md

```diff
@@ -11,6 +11,17 @@ To convey the intended message effectively in a diagram, follow these basic prin
 
 For details on how to format diagrams and their elements in Kyma documents, see the particular document sections.
 
+## Reference
+
+Always add an alternative text that concisely describes the content or function of the diagram you are referring to. The alt text:
+
+- Helps to maintain accessibility for every visitor, including people with vision impairments.
+- Appears in place of a diagram if it fails to load.
+- Improves the SEO of the website by enabling crawlers to index the diagram contents better.
```

Comment:
```suggestion
- Improves SEO of the website by enabling crawlers to index the diagram contents better.
```

Reply:
I would stick to `the SEO` - it has move search results on UK and US sites.

Reply:
I also checked that and I've mostly seen results with no article. Check out this [wiki article](https://en.wikipedia.org/wiki/Search_engine_optimization) about SEO for example - here SEO also appears with no article.